docs/build/isc/v1.1/docs/how-tos/core-contracts/nft/get-nft-in-collection.mdx

```diff
@@ -0,0 +1,108 @@
+---
+description: How to get the L2 NFTs in a collection.
+image: /img/logo/WASP_logo_dark.png
+tags:
+  - EVM
+  - how-to
+  - native tokens
+  - L2 NFTs
+  - Collection
+---
+import GetNftMetadata from '../../../_partials/how-tos/token/_get-nft-metadata.md';
+
+#  Get NFTs of a Given Collection
+
+This guide will show you how to get the L2 NFTs of a given collection owned by an account using the [`ISCAccounts.getL2NFTsInCollection`](../../../reference/magic-contract/ISCAccounts.md#getl2nftsincollection) function from the [`ISCAccounts.getL2NFTsInCollection`](../../../reference/magic-contract/ISCAccounts.md) [magic contract](../../../reference/magic-contract/introduction.md).
+
+<GetNftMetadata />
+
+## Understanding the `getL2NFTsInCollection` Function
+
+The `getL2NFTsInCollection` function is a view function provided by the `ISCAccounts` interface. It takes two parameters: an `ISCAgentID` representing the owner of the NFTs, and an `NFTID` representing the collection. It returns an array of `NFTID` objects, representing the NFTs in the specified collection owned by the agent.
+
+## Why Retrieve NFTs in a Collection?
```

Comment:
Same as in the other PR. I would remove this section and the Prerequisites

Reply:
Removed